hw2/04-background.js

```diff
@@ -1 +1,42 @@
-// Add your code here
+const body = document.getElementById("viewport");
+const timerValue = document.getElementById("timerInput");
+const timerButton = document.getElementById("button");
+let interval = null;
+let timerOn = false;
+
+function generateColorPalette() {
```

Comment:
I like the way you used color assigning . By using this we can generate wide variety of colors, instead of assigning each and every color.

Reply:
Thank you! :)
I have update the palette to use hsla instead f rgb as suggested by Kobe's comment.